### PR TITLE
Contribution2 : Fixed Grammatical error

### PR DESCRIPTION
**Pull request by bolarinwa-ajayi1**
_Please check the following:_

-   [ ] Do the tests still pass?[^1]
-   [ ] Does the code comply with the style guide?
    -   [ ] Run `make lint` from the Wagtail root.
-   [ ] For Python changes: Have you added tests to cover the new/fixed behaviour?
-   [ ] For front-end changes: Did you test on all of Wagtail’s supported environments?[^2]
    -   [ ] **Please list the exact browser and operating system versions you tested**:
    -   [ ] **Please list which assistive technologies [^3] you tested**:
-   [ ] For new features: Has the documentation been updated accordingly?

**Please describe additional details for testing this change**.

[^1]: [Development Testing](https://docs.wagtail.org/en/latest/contributing/developing.html#testing)
[^2]: [Browser and device support](https://docs.wagtail.org/en/latest/contributing/developing.html#browser-and-device-support)
[^3]: [Accessibility Target](https://docs.wagtail.org/en/latest/contributing/developing.html#accessibility-targets)

### CODE REVIEW

File: docs/getting_started/the_zen_of_wagtail.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ The key to using Wagtail effectively is to recognise that there are multiple rol
 
 A common mistake is to push too much power and responsibility into the hands of the content author and site administrator - indeed, if those people are your clients, they'll probably be loudly clamouring for exactly that. The success of your site depends on your ability to say no. The real power of content management comes not from handing control over to CMS users, but from setting clear boundaries between the different roles. Amongst other things, this means not having editors doing design and layout within the content editing interface, and not having site administrators building complex interaction workflows that would be better achieved in code.
 
-## A CMS should get information out of an editor's head and into a database, as efficiently and directly as possible.
+## A CMS should get information out of an editor's head and into a database as efficiently and directly as possible.
```
Comment (gasman) [inline docs/getting_started/the_zen_of_wagtail.md:17]: Disagree - the comma here is important. We're presenting two distinct ideas - this is the job of a CMS, AND it should do it efficiently.

Comment (bolarinwa-ajayi1) [reply docs/getting_started/the_zen_of_wagtail.md:17]: Noted,  I understand your perspective better.

File: docs/getting_started/tutorial.md [hunk 2]
```diff
@@ -870,6 +870,6 @@ Finally, we can update the `blog_page.html` template to display the categories:
 
 ## Where next
 
--   Read the Wagtail [topics](../topics/index) and [reference](../reference/index) documentation
--   Learn how to implement [StreamField](../topics/streamfield) for freeform page content
--   Browse through the [advanced topics](../advanced_topics/index) section and read [third-party tutorials](../advanced_topics/third_party_tutorials)
+-   Read the Wagtail [topics](../topics/index) and [reference](../reference/index) documentation.
+-   Learn how to implement [StreamField](../topics/streamfield) for freeform page content.
+-   Browse through the [advanced topics](../advanced_topics/index) section and read [third-party tutorials](../advanced_topics/third_party_tutorials).
```
Comment (lb-) [inline docs/getting_started/tutorial.md:875]: These lists do not need a period at the end

File: docs/getting_started/tutorial.md [hunk 3]
```diff
@@ -119,15 +119,15 @@ When logged into the admin site, a superuser has full permissions and is able to
...
+After running the above command,the server generates a URL <http://127.0.0.1:8000>. Copy and paste the link into your browser tab, which displays the welcome page.
 
 ![Browser screenshot of "Welcome to your new Wagtail site!" page, with teal egg above the title, and links to different resources. The page is shown inside a browswer tab, with browser URL bar at the top](../_static/images/tutorial/tutorial_1.png)
 
 ```{note}
 Throughout this tutorial we will use `http://127.0.0.1:8000` but depending on your setup, this could be `http://localhost:8000/` or a different IP address or port. Please read the console output of `manage.py runserver` to determine the correct url for your local site.
 ```
 
-You can now access the administrative area at <http://127.0.0.1:8000/admin>
+You can now access the administrative section by clicking on the **Admin Interface** on the welcome page at port <http://127.0.0.1:8000/admin>.
```
Comment (lb-) [inline docs/getting_started/tutorial.md:130]: I don't think we need this change, it's just becoming too wordy, also technically the port is the wrong term here. 

But let's just remove this change.

File: docs/getting_started/tutorial.md [hunk 4]
```diff
@@ -119,15 +119,15 @@ When logged into the admin site, a superuser has full permissions and is able to
 python manage.py runserver
 ```
 
-If everything worked, <http://127.0.0.1:8000> will show you a welcome page:
+After running the above command,the server generates a URL <http://127.0.0.1:8000>. Copy and paste the link into your browser tab, which displays the welcome page.
```
Comment (lb-) [inline docs/getting_started/tutorial.md:122]: I do not think we need this change, hopefully someone doing this tutorial knows how URLs and a browser work. 

We should avoid aiming this page too low, otherwise it will become too wordy for those that do know how a browser, URLs and the terminal works.

**Review by lb- — CHANGES_REQUESTED** [review]
Thanks @bolarinwa-ajayi1 but I think most of these changes are not necessary at this time. Unless you feel there is a critical problem to how these docs are worded. 

I recommend you just update this PR to include the changes to the following files only. 

- docs/getting_started/index.md
- docs/getting_started/demo_site.md

Please update your patch branch with new commits/ revised commits with the change and try to avoid opening a new PR. 

Thanks again for wanting to contribute, it's really appreciated.

File: docs/getting_started/tutorial.md (hunk 4 above)
Comment (bolarinwa-ajayi1) [reply docs/getting_started/tutorial.md]: Thanks for the feedback

File: docs/getting_started/tutorial.md (hunk 3 above)
Comment (bolarinwa-ajayi1) [reply docs/getting_started/tutorial.md]: Thanks for the feedback

File: docs/getting_started/tutorial.md (hunk 2 above)
Comment (bolarinwa-ajayi1) [reply docs/getting_started/tutorial.md]: Thanks for the feedback.

**Review by lb- — APPROVED** [review]
Thanks for fixing the other items up.

Just a note for your future PRs
* Please use a more descriptive title, something like 'Update grammar in the Documentation getting started pages', no need to say things like 'Contribution1,2,3'. This helps reviewers understand quickly what this PR is about.
* Please also include a bit of a description and take a look at the template content (checkboxes), providing a bit of a description and reasoning of the PR really helps.
